src/main/java/dev/j3fftw/litexpansion/Events.java

```diff
@@ -68,6 +74,26 @@ public void onHungerDamage(EntityDamageEvent e) {
         }
     }
 
+    @EventHandler
+    public void onBlockBreak(BlockBreakEvent e) {
+        Player p = e.getPlayer();
+        if (Constants.MACHINE_BREAK_REQUIRES_WRENCH
+            && (!Items.WRENCH.isSimilar(p.getInventory().getItemInMainHand()))
+        ) {
+            Block block = e.getBlock();
+            SlimefunItem slimefunBlock = BlockStorage.check(block);
+
+            if (slimefunBlock instanceof EnergyNetComponent
+                && !slimefunBlock.getID().endsWith("_CAPACITOR")
```

Comment:
this is a very unsafe way to do this... Just do `instanceof Capacitor` instead of relying on the id format.